spec/controllers/collaborations_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe CollaborationsController, type: :request do
+  before do
+    @author = FactoryBot.create(:user)
+    @project = FactoryBot.create(:project, author: @author)
+  end
+
+  # TODO - fix test
+  # Throws ActionController::UnknownFormat
```

Comment:
?

Reply:
This was fixed already, forgot to remove the comment. 😓